README.md

```diff
@@ -50,9 +50,12 @@ services:
         volumes:
             - /path/to/data:/config
             - /path/to-custom-ssl-keys:/config/keys
-        image: lscr.io/linuxserver/speedtest-tracker:latest
+        image: lscr.io/linuxserver/speedtest-tracker:0.20.5
         restart: unless-stopped
 ```
+## Image version
+
+Check [here](https://fleet.linuxserver.io/image?name=linuxserver/speedtest-tracker) to see what the latest version is. Due to breaking changes we don't advise to use the `latest` tag. Best pratice is to pin it to a version.
```

Comment:
Do you mind simplifying this to just

"A full list of released versions can be found here: lsio link"

Reply:
Done!